axelor-account/src/main/resources/reports/Invoice.rptdesign

```diff
@@ -5334,7 +5334,7 @@ else if(row["title_select"] == "4"){'Prof.'}</value>
                         </list-property>
                         <property name="contentType">html</property>
                         <text-property name="content"><![CDATA[<VALUE-OF>if(row._outer["partner_type_select"] != "1"){
-			if(params["Locale"].value != 'en') {
+			if(params["Locale"].value != 'en_US') {
```

Comment:
no you have to extract the language value in the Locale, it has to work even for en_UK for example